index.html

```diff
@@ -2043,46 +2048,57 @@ <h3>Lifecycle Details</h3>
 <a>verifiable presentation</a>.
           </li>
           <li>
-A <a>verifier</a> <a>verifies</a> the authenticity of
-the presented <a>verifiable presentation</a> and/or
-<a>verifiable credentials</a>. This should include checking the
-<a href="#status">credential status</a> for revocation
+A <a>verifier</a> <a>verifies</a> the authenticity of the presented
+<a>verifiable presentation</a> or <a>verifiable credentials</a>. This should
```

Comment:
```suggestion
<a>verifiable presentation</a> and <a>verifiable credentials</a>. This should
```

This use to be `and/or` ... and in this edit it was simplified to `or`. But if we're simplifying, I think it would be better to simplify to `and` as the much more common case.